SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec

```diff
@@ -20,12 +20,10 @@ Source0:        https://github.com/Azure/application-gateway-kubernetes-ingress/
 #           --mtime="2021-04-26 00:00Z" \
 #           --owner=0 --group=0 --numeric-owner \
 #           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
-#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#           -cf %%{name}-%%{version}-%%{release}-vendor.tar.gz vendor
```

Comment:
With the script, this set of steps isn't necessary anymore. If you do wish to keep it, though, `-cf` should be `-czf`.

Reply:
I removed it as in your docker-compose example